apps/emqx/src/emqx_authentication.erl

```diff
@@ -314,6 +314,10 @@ do_authenticate([#authenticator{provider = Provider, state = State} | More], Cre
             %% {continue, AuthData, AuthCache}
             %% {error, Reason}
             {stop, Result}
+    catch
+        error:Reason:Stacktrace ->
+            ?LOG(warning, "The following error occurred in '~s' during authentication: ~p", [ID, {Reason, Stacktrace}]),
```

Comment:
use `?SLOG`.
e.g.

```
catch
    Exception : Reason : Stacktrace ->
        ?SLOG(warngin, "auth_provider_error", #{exception => Error, reason => Reason, stacktrace => Stacktrace, provider_id => ID})
```

Reply:
Should our log content be more readable?